pkg/apis/hive/v1/dnszone_types.go

```diff
@@ -133,6 +133,9 @@ const (
 	ZoneAvailableDNSZoneCondition DNSZoneConditionType = "ZoneAvailable"
 	// ParentLinkCreatedCondition is true if the parent link has been created
 	ParentLinkCreatedCondition DNSZoneConditionType = "ParentLinkCreated"
+	// MissingDomainClientCondition is true if we try to reconcile a DNSZone and the controller
+	// cannot find a nameServerScraper/nameServer Query client
+	MissingDomainClientCondition DNSZoneConditionType = "MissingDomainClient"
```

Comment:
As an end user, I don't think that I would know what is meant be "MissingDomainClient".